excelutils/excel_sbol_utils/library2.py

```diff
@@ -119,7 +123,7 @@ def subcomponents(rowobj):
 			rowobj.obj.variableComponents.add(var_comp)
 
 	else:
-		raise KeyError(f'The object type "{type(rowobj.obj)}" does not allow subcomponents. (sheet:{self.sheet}, row:{self.sht_row}, col:{self.sht_col})')
+		raise KeyError(f'The object type "{type(rowobj.obj)}" does not allow subcomponents. (sheet:{rowobj.sheet}, row:{rowobj.sht_row}, col:{rowobj.sht_col})')
```

Comment:
rowobj has no attribute sht_col. You can give rowobj.col_cell_dict instead (this is because we no longer go by column but by sbol term so you can't say the exact column that was being used caused an issue

Reply:
Ok, I'll change that.